packages/generators/utils/add/questions/index.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	AutoComplete,
```

Comment:
Could you rephrase your questions to the ones that are on master right now?

Reply:
Questions are the same as in the master